Src/Newtonsoft.Json.UnityConverters.Tests/ConvertingUnityTypes/TypeTester.cs

```diff
@@ -30,12 +68,18 @@ protected virtual bool AreEqual(T a, T b)
             return a.Equals(b);
         }
 
-        private void AssertAreEqual(T expected, T actual)
+        protected virtual string ToString(T value)
+        {
+            return value?.ToString() ?? "<null>";
+        }
+
+        protected void AssertAreEqual(T expected, [MaybeNull] T actual, string? message = null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5420094)